File: C/Cap04/c04ex01.c
// c04ex01.c

#include <stdio.h>

int main(void) {
  int A, B, X;

  printf("Entre o 1o. valor numerico: ");
  scanf("%d", &A);
  while (getchar() != '\n'); 

  printf("Entre o 2o. valor numerico: ");
  scanf("%d", &B);
  while (getchar() != '\n'); 

  X = A + B;

  if (X > 10)
    printf("Resultado = %d\n", X);

  printf("\nTecle <Enter> para encerrar... ");
  while (getchar() != '\n');

  return 0;
}

File: DART/Cap04/c04ex01.dart
// c04ex01.dart

import 'dart:io';

void main() {
  int A;
  int B;
  int X;

  stdout.write("Entre o 1o. valor numerico: ");
  A = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

  stdout.write("Entre o 2o. valor numerico: ");
  B = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

  X = A + B;

  if (X > 10)
    print("Resultado = $X");

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
